cmd/sovereignnode/incomingHeader/incomingHeaderProcessor_test.go

```diff
@@ -363,26 +491,49 @@ func TestIncomingHeaderHandler_AddHeader(t *testing.T) {
 
 	args := createArgs()
 
+	endpoint := []byte("endpoint")
+
 	addr1 := []byte("addr1")
 	addr2 := []byte("addr2")
 
+	token1 := []byte("token1")
+	token2 := []byte("token2")
+
+	token1Nonce := make([]byte, 0)
+	token2Nonce := []byte{0x04}
+
 	gasLimit1 := uint64(45100)
 	gasLimit2 := uint64(54100)
 
+	func1 := []byte("func1")
+	func2 := []byte("func2")
+	arg1 := []byte("arg1")
+	arg2 := []byte("arg2")
+
+	token1Data := createTokenData(big.NewInt(100))
+	token1DataBytes, err := getTokenDataBytes(token1Nonce, token1Data)
+	require.NoError(t, err)
+	token2Data := createTokenData(big.NewInt(50))
+	token2DataBytes, err := getTokenDataBytes(token1Nonce, token2Data)
+	require.NoError(t, err)
+	nftData := createNftTokenData()
+	nftDataBytes, err := getTokenDataBytes(token2Nonce, nftData)
+	require.NoError(t, err)
+
 	scr1 := &smartContractResult.SmartContractResult{
 		Nonce:    0,
 		Value:    big.NewInt(0),
 		RcvAddr:  addr1,
 		SndAddr:  core.ESDTSCAddress,
-		Data:     []byte("MultiESDTNFTTransfer@02@746f6b656e31@04@64@746f6b656e32@@32@66756e6331@61726731@61726732"),
+		Data:     []byte("MultiESDTNFTTransfer@02@" + hex.EncodeToString(token1) + "@" + hex.EncodeToString(token1Nonce) + "@" + hex.EncodeToString(token1DataBytes) + "@" + hex.EncodeToString(token2) + "@" + hex.EncodeToString(token2Nonce) + "@" + hex.EncodeToString(nftDataBytes) + "@" + hex.EncodeToString(func1) + "@" + hex.EncodeToString(arg1) + "@" + hex.EncodeToString(arg2)),
```

Comment:
Please split in multiple lines here and below.
Also, this test will be refactored